docs/publishers/cli.md

```diff
@@ -1,69 +1,86 @@
-# DPM: The Data Package Manager CLI
+# DATA: The Data Package Manager CLI
 
 - [Getting started](#getting-started)
 - [Installation](#installation)
 - [Commands](#commands)
 - [Configuration](#configuration)
 - [Usage](#usage)
     - [Publish](#publish)
-    - [Tag](#tag)
+    - [Download](#download)
     - [Delete](#delete)
-    - [Undelete](#undelete)
+    - [Information](#information)
+    - [Normalize](#normalize)
+    - [Validate](#validate)
+    - [Configuration](#configuration)
+    
 - [Links](#links)
 
 ## Getting started
 
-The dpm is a command-line tool aimed to help publishers to prepare and upload data to the DataHub. With dpm you will be able to:
+The data is a command-line tool aimed to help publishers to prepare and upload data to the DataHub. With data you will be able to:
 
+* Publish Data Package to DataHub
+* Get Data Package from DataHub
+* Remove uploaded Data Package from DataHub
+* Get information about particular Data Package
+* Normalize Data Package according to the specs
 * Validate your data to ensure its quality
-* Publish Data Package
-* Tag uploaded Data Package to create historical snapshot
-* Remove uploaded Data Package that is no longer needed
+* Set up configuration file in order to publish
 
 ## Installation
 
-You can install unstable version directly from the code repository:
+### Installing binaries without npm
 
+On the [releases](https://github.com/datopian/datahub-cli/releases) page, you can download pre-built binaries for MacOS and LinuxOS x64. You need to put pre-built binary in bin directory, your path eg: `/usr/local/bin/`.
```

Comment:
You may need to put the pre-built binary in the `bin` directory (e.g.: `/usr/local/bin/`).